content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+
+The more users we have in Nightly, the more stable future versions of
+Firefox become.
+
+If everybody waits to use the last release of Firefox, it means
+that Developer Edition and Beta steps don't make any sense because
+they will not be used and bugs that might occur will appear directly
+at the Release stage, making a poor Firefox experience for people that
+don't even know what a browser is.
+
+By using Nightly, you will help us make a really great Firefox and as
+a benefit you will have this great Firefox experience 3 months ahead
+of schedule!
+
+Also most of the time, if you discover a problem with your Firefox
+Nightly, it will be fixed in the next release the day after, while
+with Firefox Release you will potentially have to wait for 6 weeks.
+
+Help add-ons developers to validate that their add-ons is still working
+-----------------------------------------------------------------------
+
+Firefox is transitionning from a complex and error prone way of
+writing add-ons to a standardized way of writing Web extensions, which
+are most likely to be compatible with Chrome and use well known Web
+technologies which makes them easier to develop.
+
+This is challenging for add-ons creators that already have a working
+add-on that needs to be rewritten.
+
+As a community we can help them. If they already have a Google Chrome
+add-on, it is most likely that they will be able to install it in
+Firefox directly. Plus, Firefox exposes more add-ons API
+than Chrome and we will create more in 2017 to help as much as
+possible port previous add-ons to the Web extensions standard.
+
+As a user we can help to test these new add-ons to keep a good
+experience for our users.
+
+Have a better developer experience to develop your Web apps
+-----------------------------------------------------------
+
+If you develop Web applications you might have had some problems to
+debug them with Firefox.
+
+We have a full team of developers dedicated to make Firefox the best
+tool for developping Web applications. This team is improving Web
+developer tools everyday and by using Nightly it will make your life
+easier.
+
+I wish that at some point you would even want to open Firefox because
+it is easiser than debugging your app in Chrome. And I have good hope
+that we reach this point in 2017.
+
+
+Give a really valuable contribution to the Mozilla project without coding
+-------------------------------------------------------------------------
+
+Often people ask me how they can contribute to the Mozilla Firefox
+project.
+
+To be honest, starting to use Nightly today is a real huge step toward
+that goal.
+
+To use Firefox Nightly as a daily user is a simple and awesome way to
+help Firefox because it gives us feedback on the work we are doing
+ahead of time and also everyday. We don't have to wait for 18 weeks to
+get feedback on our work, we can have it the day after.
+
+By the time I write this article we have only few handfuls of users
```

Comment:
Instead of `By the time I write this article`, the standard (slightly formal) way to express this is `As of this writing`, (although [some prefer](http://english.stackexchange.com/questions/199381/is-at-the-time-of-writing-correct#199392) `At the time of writing [this article]`).